pkg/kubelet/reason_cache.go

```diff
@@ -40,8 +40,8 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// Reason is the cached item in ReasonCache
-type reasonItem struct {
+// ReasonItem is the cached item in ReasonCache
+type ReasonItem struct {
```

Comment:
why should this be exported?

Reply:
Great question - its the return type of a public method, so I think either the method needs to be private or the type needs to be public.

I chose the latter because I'd like to remove pkg/kubelet/reason_cache.go out of
the kubelet package and into its own package. This diff is a precursor
for those efforts.

Reply:
ah,  SGTM
the other approach would be making the return an interface, I'm not familiar enough with these internals to verify this, will CC